lib/profiles/TR/FPIG-NOTE.js

```diff
@@ -3,6 +3,7 @@ const ignote = require('./IG-NOTE');
 
 const config = { ...ignote.config };
 config.previousVersion = false;
+config.longStatus = 'First Public Interest Group Note';
```

Comment:
As mentioned in the issue (https://github.com/w3c/specberus/issues/1251#issuecomment-847521744), this will require more discussions to make such changes.
Instead, rely on the `previousVersion` config.